database/migrations/2024_05_30_094235_add_language_id_to_questions_and_answers.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('questions', function (Blueprint $table) {
+            $table->foreignId('language_id')->nullable()->constrained('languages')->after('quiz_id');
+        });
+
+        Schema::table('answers', function (Blueprint $table) {
+            $table->foreignId('language_id')->nullable()->constrained('languages')->after('question_id');
+        });
+    }
+
```

Comment:
add transaction in the up and down function since you have two insertation in two different tables